### ISSUE TITLE
WIKI - Application Process for Suppliers / Travel Agents

### ISSUE THREAD

**Issue by sebas-led** [toursbylocals/wiki#76]
**Summary**
Suppliers, Affiliates, Agencies, and any other type of user that would eventually have access to the kitchen would have a process to get there. In order to unify the processes, we need a way to structure it so that first we create an account and then ask for the specific details. In addition, we are adding KYC for v3, which needs to be done prior to the account being created.

**ACS**

- [x] Document the process to apply as a Guide in v2 & v3 (already done, just consolidating documents into this one)
- [x] Document the process to apply as a Travel Agency in v2 & v3
- [x] Document the process to apply as an Affiliate in v2 & v3

**Comment by sebas-led** [comment]
https://docs.google.com/document/d/1i2akWrkcw1zYSOdWgGUGdXFO4ZVeWw_HHz7mCQ3SRFs/edit#

**Comment by codedavinci** [comment]
Looking good
